bin/mink-test-server

```diff
@@ -9,5 +9,8 @@ base_dir=$(dirname "$(dirname "$(readlink -f "$0")")")
 host=${MINK_HOST-localhost:8002}
 php_bin=${MINK_PHP_BIN-php}
 
+# see why USE_ZEND_ALLOC=0 is required, https://bugs.php.net/bug.php?id=71443
```

Comment:
this bug is fixed since years in PHP. If that is the reason why we have to use `USE_ZEND_ALLOC=0`, then this is weird

Reply:
a php process was dying with OOM, I'll have to re-check.
Here are some references:
- https://wildwolf.name/php-7-2-and-valgrind/
- https://bugs.php.net/bug.php?id=75683

Reply:
hmm, this bug is referenced by a merge commit from March 2018, rather than something fixed in 2016. So it might be a better candidate already.

Reply:
it's quite weird, but it's no longer relevant.